apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/CollectionAssertionSamples.kt

```diff
@@ -0,0 +1,83 @@
+//TODO remove file with 1.0.0
+@file:Suppress("DEPRECATION")
+
+package ch.tutteli.atrium.api.infix.en_GB.samples.deprecated
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.samples.fails
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.translations.DescriptionComparableAssertion
+import kotlin.test.Test
+
+class CollectionAssertionSamples {
+    private val isLessThanDescr = DescriptionComparableAssertion.IS_LESS_THAN.getDefault()
+    private val isGreaterThanDescr = DescriptionComparableAssertion.IS_GREATER_THAN.getDefault()
+
+    @Test
+    fun isEmpty() {
+        expect(listOf<Int>()) toBe empty
+
+        fails {
+            expect(listOf(1, 2, 3)) toBe empty
+        }
+    }
+
+    @Test
+    fun isNotEmpty() {
+        expect(listOf(1, 2, 3)) notToBe empty
+
+        fails {
+            expect(listOf<Int>()) notToBe empty
+        }
+    }
+
+    @Test
+    fun hasSize() {
+        expect(listOf(1, 2, 3)) hasSize 3
+
+        fails {
+            expect(listOf(1, 2, 3)) hasSize 1
+        }
+    }
+
+    @Test
+    fun sizeFeature() {
+        expect(listOf(1, 2, 3)) size {
+            it isGreaterThan 1 // subject is now of type Int (actually 3)
+        } size {
+            it isLessThan 4 // subject is still of type Int (still 3)
+        }
```

Comment:
here we want to see the overload of size in action which does not expect an assertionCreator -lambda. It's a bit more tricky than in the api-fluent especially if we want to add all remarks. please rewrite it (and other places) to:
```suggestion
        expect(listOf(1, 2, 3)) {
            size /*                 subject is now of type Int (actually 3)
            */ isGreaterThan 1 /*   subject is still of type Int (still 3)
            */ isLessThan 2
        }
```
Btw. good catch that it is still 3 and not still 1 as written in the api-fluent example. Do you mind to fix it there as well?